python/libraries/times.py

```diff
@@ -7,7 +7,15 @@ async def sleep(time):
     await asyncio.sleep(time / 1000)
 
 
+async def getTime():
+    return t.time()
```

Comment:
Interesting, this returns a timestamp in seconds, while `sleep` uses milliseconds

Reply:
Wait it does?

Reply:
Fixed